source/docs/changelog.md

```diff
@@ -72,6 +74,4 @@ it('foo', function () {
 
 ## [v0.1.0 (2020-05-09)](https://github.com/pestphp/pest/commit/de2929077b344a099ef9c2ddc2f48abce14e248f)
 ### Added
-- First version
-
-Next section: [Upgrade Guide →](/docs/upgrade-guide)
```

Comment:
@octoper can you have this fixed?

Reply:
Looks like in [the changelog workflow](https://github.com/pestphp/pest/blob/master/.github/workflows/changelog.yml#L40) we need to add the following:
```

Next section: [Upgrade Guide →](/docs/upgrade-guide)
```

https://github.com/pestphp/pest/pull/115

Reply:
@nunomaduro, shall we just manually add this to this PR? 🤔